src/cli/options.rs

```diff
@@ -1185,7 +1189,7 @@ impl Options {
     }
 
     fn make_publishers_response_sc<'a, 'b>(app: App<'a, 'b>) -> App<'a, 'b> {
-        let mut sub = SubCommand::with_name("response").about("Show RFC8183 Repository Response XML");
+        let mut sub = SubCommand::with_name("response").about("Show idexchange:: Repository Response XML");
```

Comment:
You probably didn't mean to change this ;-)

Reply:
nope, fixed